sale_commission/models/account_move.py

```diff
@@ -234,3 +251,57 @@ def _skip_future_payments(self, date_payment_to):
             if any(date_payment_to < date for date in payments_dates):
                 return True
         return False
+
+    def _partial_commissions(self):
```

Comment:
**todo**: Please add a docstring explaining the purpose of this method. I find it hard to follow personally